autoload/neomake/makers/ft/elm.vim

```diff
@@ -0,0 +1,56 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#elm#EnabledMakers() abort
+    return ['elmMake']
+endfunction
+
+function! neomake#makers#ft#elm#elmMake() abort
+    return {
+        \ 'exe': 'elm-make',
+        \ 'args': ['--report=json', '%:p', '--output=' . neomake#utils#DevNull()],
```

Comment:
You do not need the `%:p`, do you?
`append_file` should append the current file.